bin/qunex_container

```diff
@@ -931,6 +977,79 @@ def main(args=None):
         subprocess.run(env_command, shell=True, check=False)
         exit(0)
 
+    # interactive mode
+    if interactive:
+        print("\n---> Starting QuNex container in interactive mode...\n")
+
+
+        # binds
+        if bind is None:
+            container_bind = f"{homedir}:{homedir}"
+        else:
+            container_bind = f"{homedir}:{homedir},{bind}"
+
+        # bind cuda path if provided
+            if cuda_path:
+                container_bind += f",{cuda_path}:/usr/local/cuda"
+
+        if apptainer:
+            container_command = []
+            container_command += ["singularity", "exec"]
+            container_command += ["--bind", container_bind]
+
+            if containeropt:
+                container_command += shlex.split(containeropt)
+
+            # no clean env
+            if nocleanenv is None:
+                container_command.append("--cleanenv")
+
+            # cuda
+            if cuda:
+                container_command += ["--nvccli", "-u"]
+
+            # nv
+            if nv:
+                container_command += ["--nv "]
```

Comment:
A lot of these parameters and also others (e.g. bind) are now set at three locations. This PR is not at fault for this as it was at already at 2 locations before. We can merge this as is, but I will then do a polish pass over the whole script to trim down on duplicate code.